network/network-tutorial-overlay.md

```diff
@@ -401,52 +399,47 @@ The example uses Linux hosts, but the same commands work on Windows.
     uw9etrdymism        test-net            overlay             swarm
     ```
 
-5.  Remember that you created `alpine1` from `host1` and `alpine2` from `host2`.
-    Now, attach to `alpine2` from `host1`:
+7.  On `host1`, ping `alpine2` within the `/bin/sh` session of `alpine1`:
```

Comment:
I think you can just say:  On `host1`, ping `alpine2`
When reading the full sentence I'm left with a couple of questions about `/bin/sh` and session that end up being more distracting than helpful.

Reply:
Simplified. Thanks for the review, Joao. I tightened some things up but I kept it to one ping.